test/functional/test_framework/test_node.py

```diff
@@ -549,11 +550,17 @@ def p2p(self):
         assert self.p2ps, self._node_msg("No p2p connection")
         return self.p2ps[0]
 
+    def num_connected_mininodes(self):
+        """Return number of p2ps that are connected to the node. Excludes test nodes."""
+        peer_subversions = [peer['subver'] for peer in self.getpeerinfo()]
+        return peer_subversions.count(MY_SUBVERSION)
```

Comment:
Alternatively, use a conditional in the list comprehension:

```
return len([peer for peer in self.getpeerinfo() if peer['subver'] == MY_SUBVERSION])
```

I think that's slightly clearer, but it's probably just a personal preference thing.